examples/autoscale/main.tf

```diff
@@ -170,77 +146,51 @@ module "autoscale" {
     dhcp-send-client-id         = "yes"
     dhcp-accept-server-hostname = "yes"
     dhcp-accept-server-domain   = "yes"
-    dns-primary                 = "8.8.8.8"
+    dns-primary                 = "169.254.169.254" # Google DNS required to deliver PAN-OS metrics to Cloud Monitoring
     dns-secondary               = "4.2.2.2"
   }
 
-  # Example of full bootstrap with Google storage bucket.
-  /*
-  metadata = {
-    mgmt-interface-swap                  = "enable"
-    vmseries-bootstrap-gce-storagebucket = "my-google-bootstrap-bucket"
-    serial-port-enable                   = true
-    ssh-keys                             = "~/.ssh/vmseries-ssh-key.pub"
-  }
-  */
-  named_ports = [
-    {
-      name = "http"
-      port = "80"
-    },
+  depends_on = [
+    module.mgmt_cloud_nat
   ]
 }
 
-
-#-----------------------------------------------------------------------------------------------
-# Internal Network Balancer to distribute traffic to VM-Series trust interfaces
-#-----------------------------------------------------------------------------------------------
-/* 
-  The load balancers are not required for this example.  It is here to provide an example
-  of how to use the load balancer modules. 
-*/
-
+# Internal Network Load Balancer to distribute traffic to VM-Series trust interfaces
 module "intlb" {
   source = "../../modules/lb_internal/"
 
-  name                = "${local.prefix}intlb"
-  network             = module.vpc_trust.network_id
-  subnetwork          = module.vpc_trust.subnets_self_links[0]
-  all_ports           = true
-  backends            = module.autoscale.backends
+  name              = "${var.name_prefix}internal-lb"
+  network           = module.vpc_trust.network_id
+  subnetwork        = module.vpc_trust.subnets_self_links[0]
+  all_ports         = true
+  health_check_port = 80
```

Comment:
Maybe we can make this a variable and worst case give a default value ?

Reply:
Let's do it as part of refactor for NGFW Hub.